src/statmodels.jl

```diff
@@ -542,6 +542,56 @@ function show(io::IO, ct::CoefTable)
     nothing
 end
 
+show(mime::MIME"text/markdown", ct::CoefTable) = show(Base.stdout, mime, ct)
+
+function show(io::IO, ::MIME"text/markdown", ct::CoefTable)
+    cols = ct.cols; rownms = ct.rownms; colnms = ct.colnms;
+    nc = length(cols)
+    nr = length(cols[1])
+    if length(rownms) == 0
+        rownms = [lpad("[$i]",floor(Integer, log10(nr))+3) for i in 1:nr]
```

Comment:
Looks like this isn't tested either?

Reply:
This code was literally copy-and-pasted from the plaintext `show` method.

Reply:
Yeah, but this could be tested by passing zero rows IIUC.